tests/run-pass/sync.rs

```diff
@@ -1,18 +1,95 @@
-// Just instantiate some data structures to make sure we got all their foreign items covered.
 // Requires full MIR on Windows.
```

Comment:
This comment is awfully outdated, these days we use full MIR everywhere anyway. Could you remove it?

Reply:
Yes, see 011541aabdbec817804ff8625697b70fb107d7c2

Reply:
No I mean the "Requires full MIR on Windows", which so far you did not remove as far as I can see.

Reply:
Ah, sorry, skimmed too fast